### ISSUE TITLE
Comments with Remote Signer

### ISSUE THREAD

**Issue by xeruf** [servuscms/servus#15]
Just found https://npub.pro and then this - so I wondered if anonymous comments support like for npub.pro could be supported here too, so people can comment on my blog with native nostr?

**Comment by ibz** [comment]
Yes, I definitely want to support that. Keeping this issue open as a reminder.

**Comment by ibz** [comment]
I think npub.pro is more mature at the moment, so if you need comments, go for that. But it is also very different in scope... What it does (as far as I know) is fetch events you post to other relays and generate a web page. Servus however stores your events in local files and also acts as a personal relay, so it's a different thing. Although the final result is the same (and npub.pro is more mature).
